iroh/src/commands/list.rs

```diff
@@ -14,6 +14,12 @@ pub enum Commands {
         #[clap(long, default_value_t = DEFAULT_RPC_PORT)]
         rpc_port: u16,
     },
+    /// List the available blobs on the running provider.
+    IncompleteBlobs {
```

Comment:
will this list incomplete collections?

Reply:
Collections are blobs, so yes.

The concept of collections does not really exist anymore except in a few places.